package.json

```diff
@@ -10,21 +10,20 @@
     "start:dev": "./node_modules/nodemon/bin/nodemon.js --trace-warnings ./server.js --config nodemon.json",
     "start": "node src/index.js",
     "migrate": "NODE_ENV=development sequelize db:migrate --config src/config/databases.js --migrations-path src/db/migrations",
-    "docker:workers": "NODE_ENV=development docker exec -it resonate-api node src/jobs/queue-worker.js run",
-    "docker:migrate": "NODE_ENV=development docker exec -it resonate-api npx sequelize db:migrate --config src/config/databases.js --migrations-path src/db/migrations",
-    "docker:migrate:undo:all": "NODE_ENV=development docker exec -it resonate-api npx sequelize db:migrate:undo:all --config src/config/databases.js --migrations-path src/db/migrations",
-    "docker:seed:all": "NODE_ENV=development docker exec -it resonate-api npx sequelize db:seed:all --config src/config/databases.js --seeders-path src/db/seeders",
-    "docker:seed:undo:all": "NODE_ENV=development docker exec -it resonate-api npx sequelize db:seed:undo:all --config src/config/databases.js --seeders-path src/db/seeders",
-    "docker:compose:test:up": "docker compose -f docker-compose.test.yml up",
-    "docker:compose:test:down": "docker compose -f docker-compose.test.yml down",
+    "docker:workers": "docker exec -it resonate-api node src/jobs/queue-worker.js run",
+    "docker:migrate": "docker exec -it resonate-api npx sequelize db:migrate --config src/config/databases.js --migrations-path src/db/migrations",
+    "docker:migrate:undo:all": "docker exec -it resonate-api npx sequelize db:migrate:undo:all --config src/config/databases.js --migrations-path src/db/migrations",
+    "docker:seed:all": "docker exec -it resonate-api npx sequelize db:seed:all --config src/config/databases.js --seeders-path src/db/seeders",
+    "docker:seed:undo:all": "docker exec -it resonate-api npx sequelize db:seed:undo:all --config src/config/databases.js --seeders-path src/db/seeders",
+    "docker:compose:test:up": "docker-compose -f docker-compose.yml -f docker-compose.test.yml up",
```

Comment:
Note the new chaining of the docker-compose files. This makes it so that anything defined in `docker-compose.test.yml` that is _already_ defined in `docker-compose.yml` takes precedence.

Reply:
Does `docker compose down` cleanly shut down both the normal and the test containers?